x-pack/plugins/security_solution/cypress/integration/exceptions/exceptions_flyout.spec.ts

```diff
@@ -130,12 +130,12 @@ describe('Exceptions flyout', () => {
     cy.get(ADD_AND_BTN).click();
     addExceptionEntryFieldValue('@timestamp', 1);
     cy.get(ADD_AND_BTN).click();
-    addExceptionEntryFieldValue('c', 2);
+    addExceptionEntryFieldValue('agent.hostname', 2);
```

Comment:
Most of the values changed on this file are incorrect. In this test, we are adding an invalid value in purpose, in this case `c`, changing that value for a valid one in the mapping, will change the purpose of the test.